package.json

```diff
@@ -5,16 +5,19 @@
   "main": "ping-centre.js",
   "license": "MPL-2.0",
   "scripts": {
-    "test": "npm-run-all test:*",
+    "bundle": "webpack",
+    "test": "npm run bundle && npm run test:node && npm run test:browser",
```

Comment:
Should this be `npm run bundle && npm-run-all test:*`?
Currently it looks like there are 2 errors if you run `npm run test:lint`:

```sh
% ping-centre git:(webpack) $ git rev-parse --short HEAD # 53c159a
% ping-centre git:(webpack) $ npm run test:lint

> ping-centre@1.0.0 test:lint /Users/pdehaan/dev/github/mozilla/ping-centre
> eslint .

/Users/pdehaan/dev/github/mozilla/ping-centre/test/karma.conf.js
  1:1  error  Use the global form of 'use strict'  strict

/Users/pdehaan/dev/github/mozilla/ping-centre/webpack.config.js
  1:1  error  Use the global form of 'use strict'  strict

✖ 2 problems (2 errors, 0 warnings)
```